{"name": "test_login_failure[invalid_credentials]", "status": "passed", "description": "测试用户登录失败", "steps": [{"name": "执行测试用例: 无效用户登录失败", "status": "passed", "steps": [{"name": "验证错误信息", "status": "passed", "start": 1750888407471, "stop": 1750888407507}], "start": 1750888407056, "stop": 1750888407507}], "parameters": [{"name": "test_case", "value": "{'id': 'invalid_credentials', 'email': 'error@example.com', 'password': 'error', 'expected_email_msg': 'Email or password is invalid.', 'expected_password_msg': 'Email or password is invalid.', 'description': '无效用户登录失败'}"}], "start": 1750888407056, "stop": 1750888407508, "uuid": "67900f80-02fe-4c26-9000-1e7bc9314fd8", "historyId": "77ee5f8af18d03135dd5f524f4797d38", "testCaseId": "3b22a26d842dad5d1bde7a7dba0be0b7", "fullName": "testcase.test_login.TestLogin#test_login_failure", "labels": [{"name": "story", "value": "登录失败"}, {"name": "feature", "value": "登录功能"}, {"name": "tag", "value": "order(2)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('driver')"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "SQG"}, {"name": "thread", "value": "13300-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}]}